app/models/chore.rb

```diff
@@ -40,8 +40,12 @@ def due_class(due_date = next_due)
   end
 
   def add_business_days(start_date:, work_days:)
-    raise ArgumentError, 'start_date must be a Date' unless start_date.instance_of?(Date)
-    raise ArgumentError, 'work_days must be a positive integer' unless work_days.is_a?(Integer) && work_days > 0
+    start_date = start_date.to_date
```

Comment:
I really don't understand your suggestion. What logic are you talking about separating out? This commit is about accepting inputs that I previously raised errors for. Are you suggesting something along the lines of the following:

```
def add_business_days(start_date:, work_days:)
  start_date, work_days = clean_bad_business_days_inputs(start_date, work_days)
```